examples/third_party/glfw/CMakeLists.txt

```diff
@@ -30,5 +30,5 @@ if(WIN32)
     add_definitions(-D_GLFW_USE_CONFIG_H)
 endif()
 
-add_library(glfw3 ${GLFW3_CPP} ${GLFW3_HPP})
+add_library(glfw3 STATIC ${GLFW3_CPP} ${GLFW3_HPP})
```

Comment:
If this is really only a Windows fix, then should it not be inside the 
` if(WIN32)`
?
```
if(WIN32)
    add_definitions(-D_GLFW_USE_CONFIG_H)
    add_library(glfw3 STATIC ${GLFW3_CPP} ${GLFW3_HPP})
else()
    add_library(glfw3 ${GLFW3_CPP} ${GLFW3_HPP})
endif()
```